### PR TITLE
Custom styling for SRO Measure report

### PR DESCRIPTION
**Pull request by tomodwyer**
Adds custom styling for navigation and details/summary sections into the SRO Measure report.

https://user-images.githubusercontent.com/24863179/148538824-c5ccd4fc-addc-41a9-8acd-2c40008597c0.mov

### CODE REVIEW

File: assets/src/scripts/_sro-measures.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        detailsEl({
+          summaryText: "Why it matters",
+          content: detailsBuilder({ element }),
+        })
+      );
+
+      element.remove();
+    });
+  });
+}
```
Comment (tomodwyer) [inline assets/src/scripts/_sro-measures.js:68]: All this will be updated/removed once the report contains the relevant classes to target with CSS.

Comment (tomodwyer) [reply assets/src/scripts/_sro-measures.js:68]: But for the moment we will keep it until the report is live.

**Comment by rebkwok** [comment]
Is this intended to be a temporary measure for the SRO report?  Do we not want to have more generic styling that can be applied to any report, rather than checking for specific report names in the path?

**Comment by tomodwyer** [comment]
@rebkwok Yes, Jon has worked on added classes to the report, which I'll be able to use once the report is live. Those classes can then be used in future reports to give the same menu and details elements.

But this is the first draft to check everything works as expected with that report.

**Review by rebkwok — APPROVED** [review]
Happy to rubber stamp, assuming the custom styling for named reports is a one-off/temporary thing

**Pull request merged**